Cargo.toml

```diff
@@ -21,7 +21,7 @@ bee-common = { git = "https://github.com/iotaledger/bee.git", branch = "dev" }
 url = { version = "2.2", features = [ "serde" ] }
 tokio = { version = "1.3", features = ["macros", "sync", "time", "rt", "rt-multi-thread"] }
 rand = "0.8"
-rusqlite = { version = "0.24", features = ["bundled"] }
+rocksdb = "0.15"
```

Comment:
By default, it will build all supported [compression algorithms](https://github.com/rust-rocksdb/rust-rocksdb#compression-support). Should we choose one and disable the rest?

Reply:
do you know which one we should use? i think zstd is pretty good, but is it the best one?

Reply:
It seems like they [recommend](https://github.com/facebook/rocksdb/wiki/Compression#configuration) LZ4 or Zstd. Zstd gives a better compression ratio but does not compress as fast as LZ4 (probably not a big deal)

Reply:
i'll go with lz4 since speed is more important here IMO